hbase-server/src/main/java/org/apache/hadoop/hbase/mob/DefaultMobStoreCompactor.java

```diff
@@ -317,7 +318,14 @@ protected boolean performCompaction(FileDetails fd, InternalScanner scanner, Cel
 
       do {
         hasMore = scanner.next(cells, scannerContext);
-        now = EnvironmentEdgeManager.currentTime();
+        currentTime = EnvironmentEdgeManager.currentTime();
+        if (LOG.isDebugEnabled()) {
+          now = currentTime;
+        }
+        if (closeChecker.isClosedByTimeLimit(store, currentTime)) {
```

Comment:
This is a bug fix?

Reply:
s/isClosedByTimeLime/isTimeLimit/ ... and s/isClosedBySizeLiimt/isSizeLimit/

Reply:
> This is a bug fix?

No, It is not a bug fix. I think it is `improvement`.

> s/isClosedByTimeLime/isTimeLimit/ ... and s/isClosedBySizeLiimt/isSizeLimit/

Okay.